Standards/CCSMB-4.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-4: Lua
+
+*Author: Oliver Cooper (oeed)*
+*Last updated: 2022-01-27*
+*Version: v1.0.0
+
+## Quick information
+
+| Information |                           |
+| ----------- | ------------------------- |
+| Version     | 1.0.0                     |
+| Type        | Plain text file format    |
+| MIME        | `text/lua`                |
+| Extensions  | `.lua`                    |
+
+## Technical details
+
+*This format is based upon [plain text](/Standards/CCSMB-2.md) and as such technical details that apply to plain
+text also apply to this format.*
+
+Lua code files should contain valid, executable Lua code. These can either be self-contained programs, APIs or Lua
```

Comment:
Potentially add something about binary code files? Or is that contained in "valid, executable Lua code"?

Reply:
Actually, looking at the full thing, it states it extends plaintext. Lua files can be binary as well, so it would probably be best to put a note in here about that.

Reply:
> Potentially add something about binary code files? Or is that contained in "valid, executable Lua code"?

Do you mean Lua bytecode?

In my opinion, I think that should be a different file format RFC, with its own file extension. Thoughts?

Reply:
Yeah, lua bytecode.

`.luab` perhaps?

Reply:
Maybe these could be separate standards?